examples/crowdfunding/test/crowdfundingTest.js

```diff
@@ -53,9 +52,9 @@ describe('Crowdfunding Tests', function () {
   const getGlobal = (key) => runtime.getGlobalState(applicationId, key);
 
   // fetch latest account state
-  function syncAccounts () {
+  function syncAccounts (escrowAddress) {
```

Comment:
I am not sure if this is the right way. What if we have multiple logic signatures?

Reply:
Then the user can modify the code, this is just for syncing accounts, this varies from user to user.
Also user can directly call in test instead of having this fucntion.

Here we are getting the escrow address after deploying escrow, therefore we get the address after deploying it, the other way would be having each line there or changing the constant.

Reply:
Updating to that way